File: JanperCab/3-Application/Dtos/Customer/CabinetMakerDto.cs
using _1_Domain;
using _1_Domain.Enum;

namespace _3_Application.Dtos.Customer
{
    public class CabinetMakerDto : CustomerDto
    {
        public CabinetMakerDto()
        {
            CustomerType = CustomerType.CabinetMaker;
        }
    }
}

File: JanperCab/5-JanperCab/ClientApp/src/app/_models/customer/CabinetMakerDto.ts
import { CustomerDto } from './CustomerDto';
import { CustomerType } from 'src/app/_enums/CustomerType';

export class CabinetMakerDto extends CustomerDto {
  constructor() {
    super();
    this.$type = '_3_Application.Dtos.Customer.CabinetMakerDto, 3-Application';
    this.customerType = CustomerType.CabinetMaker;
  }
}
